### ISSUE TITLE
Broken with css-loader >= 6.0.0

### ISSUE THREAD

**Issue by volkyeth** [RyanClementsHax/storybook-addon-next#22]
### Describe the bug

Due to [this breaking change](https://github.com/webpack-contrib/css-loader/blob/master/CHANGELOG.md#600-2021-07-14) on css-loader 6.0.0:
> for url and import options Function type was removed in favor Object type with the filter property, i.e. before `{ url: () => true }`, now `{ url: { filter: () => true } }` and before `{ import: () => true }`, now `{ import: { filter: () => true } }`

And due to the the version constraint [^6.5.1](https://github.com/RyanClementsHax/storybook-addon-next/blob/35f8d44273df2f2bdefed6e0e3d1ae7ce8648982/package.json#L45) the addon is broken because of [current css-loader config that uses old format](https://github.com/RyanClementsHax/storybook-addon-next/blob/main/src/css/webpack.ts#L25-L40)

requiring css-loader@5.2.7 (before the breaking changes) is a workaround

### Your minimal, reproducible example

the repo examples should all be failing on storybook start 🤔

### Steps to reproduce

install with css-loader >6.0.0

### Expected behavior

storybook should start

### How often does this bug happen?

Every time

### Platform

macOs

### storybook-addon-next version

1.4.2

**Comment by RyanClementsHax** [comment]
Hi! Thanks for opening this issue.

I'm having trouble recreating the issue. All of the examples work just fine for me. Could you create a minimal reproduction example?

It might help to know that I recently found out that because of some package resolutions that storybook has, [the version of css loader that's actually resolved is v3.x](https://github.com/RyanClementsHax/storybook-addon-next/issues/19#issuecomment-1034272993). This leads me to ask the following question: is this a bug that _in theory_ should happen, or is this something you're _actually seeing_? 

Btw, I agree, it's totally expected to think that this addon should break given the info you just gave.

**Comment by volkyeth** [comment]
Ok, this is more complex than I thought at first. The issue is surfaced in my project by a workaround to #23 that is using "nohoist" on the yarn workspace to keep the `node_modules` of storybook-addon-next on the same folder it's used, like so:

Doing this decouples the versions of `css-loader` required by other dependencies from the one required by storybook-addon-next (`css-loader@6.5.1`), making the error I reported manifest.

I would still say this is a bug, despite manifesting only in specific circumstances, because `storybook-addon-next` is incorrectly reporting it supports `css-loader` > 6.0.0, so if the other dependencies upgraded their css-loader version this package would break in regular scenarios too.

I'm still trying to setup a minimal reproduction scenario based on the one I created for #23, but it's becoming a ninghtmare because it seems the upstream storybook packages don't bode well with yarn workspaces too :/
